packages/ui/package.json

```diff
@@ -5,21 +5,36 @@
   "license": "MIT",
   "scripts": {
     "lint": "eslint --fix './'",
-    "clean": "rm -rf .turbo node_modules"
+    "clean": "rm -rf .turbo node_modules",
+    "storybook": "start-storybook -p 6006",
+    "build-storybook": "build-storybook"
   },
   "devDependencies": {
     "@babel/core": "^7.17.7",
     "@babel/preset-react": "^7.16.7",
     "@emotion/react": "^11.8.2",
     "@emotion/styled": "^11.8.1",
     "@mui/material": "^5.5.1",
+    "@storybook/addon-actions": "^6.4.19",
+    "@storybook/addon-essentials": "^6.4.19",
+    "@storybook/addon-links": "^6.4.19",
+    "@storybook/addons": "^6.4.19",
+    "@storybook/client-api": "^6.4.19",
+    "@storybook/react": "^6.4.19",
+    "@svgr/webpack": "^6.2.1",
     "@types/react": "^17.0.40",
+    "babel-loader": "^8.2.3",
     "eslint": "^8.11.0",
     "eslint-config-commons-ui": "workspace:*",
     "prettier": "^2.6.0",
+    "prop-types": "^15.8.1",
     "react": "^17.0.2",
     "react-dom": "^17.0.2",
-    "typescript": "^4.6.2"
+    "storybook-addon-material-ui": "^0.9.0-alpha.24",
```

Comment:
Does this work with MUI v5? I see another [package](https://www.npmjs.com/package/storybook-addon-material-ui5) forked from this just to support [MUI 5](https://github.com/alexpanov/storybook-addon-material-ui/commits/version-1)

Reply:
@kilemensi True true...storybook-addon-material-ui isn't supported by version 5 of material ui. Updated that to show add https://www.npmjs.com/package/storybook-addon-material-ui5